src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -7,11 +7,14 @@
           novalidate>
 
       <div i18n="form title"
-           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}</div>
+           class="card-header">{{ action | titlecase }} {{ resource | upperFirst }} 
+           <cd-help-text>
+           <div class="copySubheading" *ngIf="action === 'Copy'">Copy the content of a source image into the newly created destination image</div> 
+          </cd-help-text>
+          </div>
 
       <!-- Parent -->
-      <div class="form-item"
-           *ngIf="rbdForm.getValue('parent')">
+      <div class="form-item" *ngIf="rbdForm.getValue('parent')">
```

Comment:
This whole Block of code I need to delete  under Parent ??

Reply:
no, not removing them. I meant to keep it as it was before. the `ngIf` block is moved to the above line which will break the linting because html lint doesn't allow two attributes to be in same line.

Reply:
Do I remove the whole block code ?